### DISCUSSION TITLE
Is it possible to compile more then one project with Lime?

### DISCUSSION THREAD

**Discussion by moxie-coder** [openfl/lime#1650] [Q&A] [answered]
I was wondering if it's possible to compile more then one project via Lime

**Comment by player-03** [comment, marked as answer]
Lime only compiles one project per `lime build` or `lime test` command, but you can absolutely run multiple commands to build multiple projects.

First make sure that each project is in its own folder, and that they each have their own project.xml file. Open a command line window in each folder, and run `lime test <target>`.

**Reply by moxie-coder** [reply]
Alright, thanks

**Comment by moxie-coder** [comment]
forgot to close this
